Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/PatientDiscoveryAuditLog.properties

```diff
@@ -41,7 +39,6 @@ ActiveParticipant_Source.RoleIDCode.@displayName=Source
 ActiveParticipant_Dest.@UserID=https://localhost:8181/Gateway/PatientDiscovery/1_0/NhinService/NhinPatientDiscovery
 ActiveParticipant_Dest.@UserIsRequestor=false
 ActiveParticipant_Dest.@UserName=Karl Skagerberg
```

Comment:
If the comment is about UserName. As mentioned in PR description, all the scripts related to Username will be handled in FHAC-652